src/main/java/com/homeworks/entity/Skill.java

```diff
@@ -0,0 +1,77 @@
+package com.homeworks.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.PostLoad;
+import javax.persistence.PostPersist;
+import javax.persistence.PostRemove;
+import javax.persistence.PostUpdate;
+import javax.persistence.PrePersist;
+import javax.persistence.PreRemove;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import java.util.HashSet;
+import java.util.Set;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "skills")
+public class Skill extends Model{
+    @Column(name = "branch")
+    private String branch;
+
+    @Column(name = "lvl")
+    private String level;
```

Comment:
better use enum here https://vladmihalcea.com/the-best-way-to-map-an-enum-type-with-jpa-and-hibernate/